src/main/java/com/revlearn/team1/dto/course/ModuleDTO.java

```diff
@@ -0,0 +1,10 @@
+package com.revlearn.team1.dto.course;
+
+public record ModuleDTO(
```

Comment:
Can you be more descriptive with the name here? is a Request/Response, is it a GET/EDIT/CREATE

I'll read more of your code to get that answer shortly but it would be really convenient if I could just understand what its supposed to be for just from the name. something like CreateModuleRequestDTO.java

Reply:
Yes.  I will implement that.  Let me merge this and push another branch with a refactor.